### PR TITLE
feat: Rough entry skeleton

### PR DESCRIPTION
**Pull request by DafyddLlyr**
image

### CODE REVIEW

File: components/layout/EntryLayout.tsx [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+export const EntryLayout = (entry?: Entry) => (
+  <div className="bg-white z-20 text-white">
+    <EntryHeader {...entry} />
+    <EntryDetails {...entry} />
+    <div className="bg-sky-700 h-96 place-items-center grid text-2xl">
+      CHART
+    </div>
+    <div className="bg-teal-900 h-96 place-items-center grid text-2xl">
+      STATIC MAP
```
Comment (jessicamcinchak) [inline components/layout/EntryLayout.tsx:86]: Not sure if you're familiar, but was thinking that Mapbox's Static Image API might be a cool lightweight option for this ! 

https://docs.mapbox.com/api/maps/static-images/#example-response-retrieve-a-static-map-with-a-geojson-point-overlay

**Review by jessicamcinchak — APPROVED** [review]
thanks for taking this one on! super helpful to have a skeleton to work from :slightly_smiling_face:

File: components/layout/EntryLayout.tsx (hunk 1 above)
Comment (DafyddLlyr) [reply components/layout/EntryLayout.tsx:86]: This is new to me but super cool - sounds perfect.

We could maybe be clever and use this if there isn't a hero image for an entry also?

**Comment by DafyddLlyr** [comment]
Nearly there with resolving conflicts and we can merge this soon 💪

**Pull request merged**